diff --git a/dist/index.d.ts b/dist/index.d.ts
index fd13f5eaae6c07666c30d81203ad14bfe789f16e..a7462c78d8591ba8a40abfbceb0b3c795f535934 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -5,7 +5,7 @@ import { Path, Location, Point, Range } from 'slate';
 import * as slate_react_dist_utils_key from 'slate-react/dist/utils/key';
 import * as slate_react_dist_utils_dom from 'slate-react/dist/utils/dom';
 import { DOMNode, DOMPoint } from 'slate-react/dist/utils/dom';
-import { Value as Value$1 } from '@udecode/slate/dist';
+import { Value as Value$1 } from '@udecode/slate';
 import React$1 from 'react';
 import { UnknownObject, Modify } from '@udecode/utils';
 
